linkage-monitor/pom.xml

```diff
@@ -78,6 +78,11 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+	<dependency>
+	  <groupId>commons-cli</groupId>
+	  <artifactId>commons-cli</artifactId>
+	  <version>1.4</version>
```

Comment:
Suggestion: how about adding commons-cli to the root pom's dependency management section? The "dependencies" module also uses it.

Reply:
good idea, done